### PR TITLE
Es imputation sg bpm test

### PR DESCRIPTION
**Pull request by saul-pountney**
I have added BPM status message calls and queue url to the error handlers aswell as fixing testing method . 
Needs someone to look over the code and re-run the test to confirm that they pass(build, test, flake8, isort, yamllint)

### CODE REVIEW

**Review by glanvl — CHANGES_REQUESTED** [review]
Couple of changes to placement of setting the queue variable

**Review by kingmushroom — APPROVED** [review]

File: calculate_movement_wrangler.py [hunk 1]
```diff
@@ -58,6 +60,10 @@ def lambda_handler(event, context):
     logger = general_functions.get_logger()
     error_message = ""
     # Define run_id outside of try block
+
+    bpm_queue_url = None
```
[the remark this answers is not in the document]
Comment (saul-pountney) [reply calculate_movement_wrangler.py:67]: fixed the issue and commented the run_id section as well to keep continuity.

File: atypicals_wrangler.py [hunk 2]
```diff
@@ -49,6 +50,9 @@ def lambda_handler(event, context):
     error_message = ""
     logger = general_functions.get_logger()
     # Define run_id outside of try block
+
```
[the remark this answers is not in the document]
Comment (saul-pountney) [reply atypicals_wrangler.py:55]: fixed the run_id comment issue and added the comment where it was missing.

**Review by glanvl — APPROVED** [review]
Looks good.

**Review by kingmushroom — APPROVED** [review]

**Pull request merged**